README.md

```diff
@@ -48,4 +49,6 @@ Video recordings are available in the ["Data Seminars Series" playlist](https://
 [6]: https://www.youtube.com/watch?v=-ck2GN75ycA&list=PL20S5EeApOSvkewFIuz2scAEkbnBIlzYy
 [7]: https://www.youtube.com/watch?v=YSc07PTeExw&list=PL20S5EeApOSvkewFIuz2scAEkbnBIlzYy
 [8]: https://www.youtube.com/watch?v=EdvmicKME7Y&list=PL20S5EeApOSvkewFIuz2scAEkbnBIlzYy
-[9]: https://www.youtube.com/watch?v=hdcwthKcAVg&list=PL20S5EeApOSvkewFIuz2scAEkbnBIlzYy
+[9]: https://docs.google.com/presentation/d/1J_xRTvOwtYeXdfqxwD5-sAswdTOw6nNo/edit?usp=drive_web&ouid=111822247062274782499&rtpof=true
+[10]: https://drive.google.com/file/d/18kwOhBFsGedxpee0Inb7Zh_0zOtssPG4/view?usp=sharing
```

Comment:
should we put this on youtube?

Reply:
Fine with me, but I'm not sure how to do that

Reply:
ok, since Alberto is away, I may do it.